src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs

```diff
@@ -606,7 +605,7 @@ private bool IsSymbolSuggestableForFunctionArg(TexlFunction function, int argInd
                 // Case 2: Aggregate type with function manipulations or scope info.
                 isSuggestable = symbolType.CoercesTo(argType, false, false, features);
             }
-            else if (argType.IsAggregate && function.ParamTypes.Any(parm => parm.Kind == symbolType.Kind))
+            else if (argType.IsAggregate && function.ParamTypes.Any(parm => parm.Kind == symbolType.Kind) && curNode.Parent is not Syntax.RecordNode)
```

Comment:
>curNode.Parent is not Syntax.RecordNode [](http://example.com/codeflow?start=109&length=39)

Should we rather check if ChildCount is 0 and argType is agrregate and Function is connector function then directly return false?

Because for connector, Dynamic intellisense would add this aggregate suggestions at IntellisenseHelper.AddTopLevelSuggestionsForCursorType()

Reply:
Discussed offline